internal/auth/providers/providers.go

```diff
@@ -24,6 +24,23 @@ var (
 	ErrServiceUnavailable = errors.New("SERVICE_UNAVAILABLE")
 )
 
+const (
+	// AzureProviderName identifies the Azure AD provider
+	AzureProviderName = "azure"
```

Comment:
I'm not sure if we need all of these constants yet, might be good to just keep Google for now and add them as the providers have been implemented as to not confused anyone that they have been.

Reply:
Sure, that seems fine. I'm planning to do these next-ish, but they can go into another PR instead, that's fine.